src/router.tsx

```diff
@@ -172,14 +168,11 @@ export function Router() {
             <Route path="/profile" component={Profile} />
             <Route path="/chat/:id" component={Chat} />
             <Route path="/feedback" component={Feedback} />
-            <Route path="/gift" component={GiftReceive} />
             <Route path="/receive" component={Receive} />
             <Route path="/redeem" component={Redeem} />
             <Route path="/request/:id" component={RequestRoute} />
             <Route path="/scanner" component={Scanner} />
             <Route path="/send" component={Send} />
-            <Route path="/swap" component={Swap} />
-            <Route path="/swaplightning" component={SwapLightning} />
```

Comment:
Think we're only temporarily disabling these for now, should bring them back eventually